docs/doc/15-sql-functions/04-array-functions/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Array Functions'
+---
+
+Array Functions
```

Comment:
The index of array in array functions are 1-based.

Reply:
Why 1-based for index?

Reply:
It's  SQL-99 standard. Snowflake is not following this.

Reply:
This is an anti-user experience(I know ClickHouse did it), the index should be 0-based for an array.
I think snow does it right:
https://docs.snowflake.com/en/sql-reference/functions/get.html

Reply:
Docs update.

Clickhouse is also 1 based, SQL is always 1-based, JDBC driver is 1 based.